portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,21 +31,44 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
You could add tests to these servlet classes as well, if you want more practice with that. Test request mechanics is a pretty common scenario, although I realize that's covered in a later week.